doc/buildAndProgram.md

```diff
@@ -1,7 +1,7 @@
 # Build
 ## Dependencies
 To build this project, you'll need:
- - A cross-compiler : [ARM-GCC (9-2020-q2-update)](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads/9-2020-q2-update)
+ - A cross-compiler : [ARM-GCC (11.2-2022.02)](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/downloads)
```

Comment:
I think it should be more explicit here that we need a compiler with the arm 32bit eabi target. Maybe add the explicit link for linux amd64 host

```suggestion
 - A cross-compiler : [ARM-GCC eabi (11.2-2022.02)](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/downloads)
   - For an amd64 linux host [gcc-arm-11.2-2022.02-x86_64-arm-none-eabi.tar.xz](https://developer.arm.com/-/media/Files/downloads/gnu/11.2-2022.02/binrel/gcc-arm-11.2-2022.02-x86_64-arm-none-eabi.tar.xz)
```